benchmarks/JavaScript/richards.js

```diff
@@ -257,7 +257,7 @@ Scheduler.prototype.trace = function (id) {
     process.stdout.write("\n");
     this.layout = 50;
   }
-  process.stdout.write(id);
+  process.stdout.write(id.toString());
```

Comment:
This is dead code. I'd not touch it, but am fine either way.

Reply:
Sorry for the other change, this was the bug I wanted to fix. On my node the previous code gives me

> TypeError [ERR_INVALID_ARG_TYPE]: The "chunk" argument must be of type string or an instance of Buffer or Uint8Array. Received type number (123)

Reply:
Ok, thanks. Fixing it would be good then :)